builder/pkg/models/book.go

```diff
@@ -0,0 +1,9 @@
+package models
+
+//Book данные книги
```

Comment:
```suggestion
// Book данные книги
```
Дальше по коду также